packages/dashmate/src/printers/printArrayOfObjects.js

```diff
@@ -0,0 +1,50 @@
+const stripAnsi = require('strip-ansi');
+const { table } = require('table');
+
+const { OUTPUT_FORMATS } = require('../constants');
+
+const UnsupportedFormatError = require('./errors/UnsupportedFormatError');
+
+/**
+ * Prints object using specified output format
+ *
+ * @param {[Object[]]} array
+ * @param {string} format
+ */
+function printArrayofObjects(array, format) {
+  let output;
+  switch (format) {
+    case OUTPUT_FORMATS.TABLE: {
+      // Init array with headings
+      const rows = [Object.keys(array[0])];
+      array.map((obj) => rows.push(Object.values(obj)));
+
+      const tableConfig = {
+        drawHorizontalLine: (index, size) => index === 0 || index === 1 || index === size,
+      };
+
+      output = table(rows, tableConfig);
+      break;
+    }
+    case OUTPUT_FORMATS.JSON: {
+      const cleanArray = [];
+      array.forEach((outputRow, i) => {
+        const cleanRow = {};
+        Object.keys(outputRow).forEach((key) => {
+          cleanRow[key] = stripAnsi(outputRow[key]);
```

Comment:
Instead of stripping colours I would suggest just to disable them in chalk instance:
```
import {Chalk} from 'chalk';

const customChalk = new Chalk({level: 0});
```